scripts/test/sample_transmission_calculator/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Tests for Sample Transmission Calculator
+
+set(TEST_PY_FILES
+    stc_model_test.py
+    stc_presenter_test.py
+    stc_view_test.py
+    )
+
+set(PYUNITTEST_QT_API pyqt5)
+pyunittest_add_test(${CMAKE_CURRENT_SOURCE_DIR} python.scripts
```

Comment:
Minor comment: Can we put `python.scripts.sampletransmissioncalculator` here? This string is used as the prefix in the test name that ctest uses. It makes running things with `ctest R` easier if they have a more identifiable prefix.